src/docs/product/data-management-settings/server-side-sampling/index.mdx

```diff
@@ -0,0 +1,119 @@
+
+Because every project is different, you can set multiple sample rules with different sample rates per project. For example, you may need more events from high converting pages, critical API endpoints, or need to focus on latency issues from your latest release.
+
+Server-side sampling also makes your SDK and your sampling configuration largely independent of each other. This helps if you're building native applications with many releases active at a given time. 
+
+## How Sentry Samples
+
+Let's imagine for a moment that the total number of transactions the SDK observes is a bucket of rocks. The way Sentry samples transactions is similar to a person sifting rocks: by pouring the bucket of rocks into a sieve, you're allowing rocks small enough to pass through it:
+
+![Visualization of how Sentry samples from total transaction volume down to post server sampling volume](how-sentry-samples.png)
+
+In this analogy, the bucket of rocks is sifted through two sieves. Both sieves can hold the total number of rocks, but the first sieve allows more rocks to pass through because it's coarser than the second. The first sieve symbolizes the process of client-side sampling by the SDK, which, in our example, samples half of the total number of transactions the SDK observes. Symbolizing the last sieve, server-side sampling receives the half of the total transactions and then filters for a quarter of the total transactions observed.
+
+<!-- prettier-ignore-start -->
+
+| Sampling Process | Rate | Count |
+|------------------|------|-------|
+|Total transactions observed | N/A | 1000 |
+| Sentry SDK sampled | 50% | 500 |
+| Sentry servers sampled | 25% | 250 |
+
+<!-- prettier-ignore-end -->
+
+Now in reality, transactions don't vary in size like a rock, but the sieve analogy illustrates how many accepted transactions are in the end calculated.
+
+Below, you can see how an incorrect configuration of your SDK and server-side sampling rates would affect the transactions received. The sieves are reversed, and the SDK allows for a quarter of the total number of transactions to pass through. In this analogy, the coarser sieve now assigned for server-side sampling, wouldn't filter any more transactions because it's coarser than its counterpart. To summarize, the sampling rate configured for your SDK needs to be equal or greater than the rate set on Sentry servers.
+
+<!-- prettier-ignore-start -->
+
+| Sampling Process | Rate | Count |
+|------------------|------|-------|
+|Total transactions observed | N/A | 1000 |
+| Sentry SDK sampled | 25% | 250 |
+| Sentry servers sampled | 50% | 250 |
+
+<!-- prettier-ignore-end -->
+
+## Traces & Propagation of Sampling Decisions
+
+TBA
```

Comment:
How about this? @jan-auer could you please sanity check this as well? Or if you have better suggestions on how to describe this part it would be very much appreciated!

For Dynamic Sampling, we want to make sampling decisions for entire traces. Sampling transactions using transaction traces is advantageous when trying to understand transactions in the context of the global system. In this case we are not interested in seeing a particular transaction, but the whole group of transactions in a trace, so sampling should occur at the trace level.

When sampling traces, the sampling rules are based on the trace context, this means that the sampling decisions are based on the information extracted from the system that initiated the transaction.

As a mental model: The head transaction in a trace determines the trace context for all following transactions in that trace. No information can be changed, added or deleted after the first propagation. The trace context is bound to only one particular trace, and all the transactions that are part of this trace.